src/user-guide/writing-suites/configuration.rst

```diff
@@ -0,0 +1,211 @@
+.. _User Guide Configuration:
+
+Workflow Configuration
+======================
+
+Cylc suites are defined in structured, validated, :cylc:conf:`flow.cylc`
+files that concisely specify the properties of, and the relationships
+between, the various tasks managed by the suite.
+
+Here we will look at the format and content of the
+:cylc:conf:`flow.cylc` file and how to configure suites.
+
+
+.. _SuiteDefinitionDirectories:
+
+Suite Configuration Directories
```

Comment:
```suggestion
Workflow Configuration Directories
```

Reply:
I've purposfully not made this change, will need to make a full pass at the docs and code rather than patchily attempting as we go.

Also terminology isn't quite settled yet, is a Cylc workflow a "flow", but when you "re-flow" a workflow you get a "flow" with a "flow-label", is that a "flow" of the "workflow" if so what is a "run" of a "workflow", wait I know, it's a "flow" 🤦.